### DISCUSSION TITLE
Pass environment options to WebView2

### DISCUSSION THREAD

**Discussion by firai** [r0x0r/pywebview#1117] [Q&A] [answered]
EDIT: I have now filed this as an issue, and the discussion may be deleted.

---

Is it possible to pass environment options when initializing `WebView2`? Elastic overscroll for touchpads is enabled by default in Edge and thus `WebView2`, which doesn't look or feel right for desktop UI. It seems possible to turn off that flag if `WebView2` is initialized directly (https://github.com/MicrosoftEdge/WebView2Feedback/issues/3016), but I can't figure out whether it's possible to pass these flags through `pywebview`. Any suggestions?

I don't know if there are any cases where elastic overscroll is appropriate, given `pywebview`'s desktop focus. Maybe it should be disabled from `pywebview` by default?

**Comment by r0x0r** [comment, marked as answer]
I have been thinking about adding a renderer specific flags as a parameter to pywebview, but haven't arrived at a crossplatform solution. Suggestions welcomed.

You may set this option via an environment variable though. See 
https://stackoverflow.com/questions/73272734/how-to-run-webview2-with-browser-flags
